### DISCUSSION TITLE
Welcome to konturno.github.io Discussions!

### DISCUSSION THREAD

**Discussion by theo-armour** [konturno/konturno.github.io#1] [Announcements]
@dashdotdotdashdotdot

It was really nice talking with you just now

I look forward to helping you get to Helsinki next year!

Let's discuss how we can collaborate:

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dashdotdotdashdotdot** [comment]
Hey Theo, great talking with you too.
